powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1

```diff
@@ -1400,6 +1400,218 @@ function UpdatePolicyEnvironmentsForTeams
+
+    $environment = Get-AdminPowerAppEnvironment -EnvironmentName $EnvironmentId
+    if ($environment -eq $null)
+    {
+        Write-Host "No environment was found with the given id."
+        return
+    }
+
+    $governanceConfiguration = $environment.Internal.properties.governanceConfiguration
+    $governanceConfiguration = CoalesceGovernanceConfiguration -GovernanceConfiguration $governanceConfiguration
+    if ($governanceConfiguration.protectionLevel -eq "Standard")
+    {
+        Write-Host "The specified environment is already managed."
+        return
+    }
+
+    $governanceConfiguration.protectionLevel = "Standard"
+    
+    $response = Set-AdminPowerAppEnvironmentGovernanceConfiguration -EnvironmentName $EnvironmentId -UpdatedGovernanceConfiguration $GovernanceConfiguration
+    if ($response.Code -ne 202)
+    {
+        Write-Host "Failed to enable Management for the specified environment."
+        Write-Host $response.Internal.Message
+        return
+    }
+    
+    Write-Host "Enabled Management for the specified environment."    
+}
+
+function DisableManagementForEnvironment
+{
+    <#
+     .SYNOPSIS
+     Disables management for the given environment.
+     .DESCRIPTION
+     The DisableManagementForEnvironment cmdlet enables management for the given environment by updating its governance configuration.
+     Use Get-Help DisableManagementForEnvironment -Examples for more details.
+     .PARAMETER EnvironmentId
+     The id (usually a GUID) of the environment.
+     .EXAMPLE
+     DisableManagementForEnvironment -EnvironmentId 8d996ece-8558-4c4e-b459-a51b3beafdb4
+     Disables management for environment with id 8d996ece-8558-4c4e-b459-a51b3beafdb4.
+    #>
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [string]$EnvironmentId
+    )
+
+    Write-Host "Retrieving environment."
+
+    $environment = Get-AdminPowerAppEnvironment -EnvironmentName $EnvironmentId
+    if ($environment -eq $null)
+    {
+        Write-Host "No environment was found with the given id."
+        return
+    }
+
+    $governanceConfiguration = $environment.Internal.properties.governanceConfiguration
+    $governanceConfiguration = CoalesceGovernanceConfiguration -GovernanceConfiguration $governanceConfiguration
+    if ($governanceConfiguration.protectionLevel -ne "Standard")
+    {
+        Write-Host "The specified environment is not managed."
+        return
+    }
+
+    $governanceConfiguration.protectionLevel = "Basic"
+    
+    $response = Set-AdminPowerAppEnvironmentGovernanceConfiguration -EnvironmentName $EnvironmentId -UpdatedGovernanceConfiguration $GovernanceConfiguration
+    if ($response.Code -ne 202)
+    {
+        Write-Host "Failed to disable Management for the specified environment."
+        Write-Host $response.Internal.Message
+        return
+    }
+    
+    Write-Host "Disabled Management for the specified environment."    
+}
+
+function IncludeInsightsForManagedEnvironmentInWeeklyEmailDigest
```

Comment:
IncludeInsightsForManagedEnvironmentsInWeeklyEmailDigest (plural for Environments)